pkgs/applications/misc/swappy/default.nix

```diff
@@ -0,0 +1,44 @@
+{ stdenv, fetchFromGitHub
+, meson
+, ninja
+, wayland
+, cairo
+, pango
+, gtk
+, pkgconfig
+, cmake
+, scdoc
+, libnotify
+, gio-sharp
+, glibc
+, glib
+}:
+
+stdenv.mkDerivation rec {
+  name = "swappy-${version}";
+  version = "1.2.0";
+
+  src = fetchFromGitHub {
+    owner = "jtheoof";
+    repo = "swappy";
+    rev = "v${version}";
+    sha256 = "13ky6nx6lprsiza62syqrq63i78fcg74lzd9pwa4kabjnqc350j5";
+  };
+
+  nativeBuildInputs = [ glibc.dev ];
+
+  buildInputs = [ meson ninja pkgconfig cmake scdoc gio-sharp libnotify gtk pango cairo wayland];
```

Comment:
> build time only dependencies are declared in `nativeBuildInputs`
https://nixos.org/nixpkgs/manual/#reviewing-contributions-new-packages

So I guess at least: meson, ninja, pkgconfig, cmake. Not sure about the others.

and it looks like "glibc.dev" can be removed;